Scripts/aws_stack_provision.ps1

```diff
@@ -1,103 +1,302 @@
 ## Powershell script for
 
```

Comment:
Closing '}' must be on a newline.
Tabbing is way too large. Must be 3. And they must be 3 spaces. Using Notepad is not a good idea. VS Code is far better.

Reply:
I will change it